tests/test_utils/test_utils.py

```diff
@@ -9,96 +9,29 @@
 import unittest
 from shutil import rmtree, copytree
 
-import tmms.utils.core_utils as TmmsUtils
+from tmms.utils import utils
```

Comment:
I really really don't like this, it lets a boring "utils" word loose in the source.  I really really think you should restore "as TmmsUtils" or "Tutils" or.....

Reply:
but didn't we decide to move from TmmsUtils to just utils...?

Reply:
Not my recollection.  I don't like simple words that are highly likely to be easily used by other code modules.  "utils" is one such thing, "library" is another.